numpy/core/getlimits.py

```diff
@@ -284,6 +289,7 @@ def _register_known_types():
                              minexp=-1022,
                              maxexp=1024,
                              it=105,
+                             it_2=106,
```

Comment:
Can we just call it `mant_dig` here also?  It would also be good to add this to `MarchAr`, until such a time where that functionality doesn't exist anymore (even if it should not really be used; besides plausibly in valgrind and we should maybe work-around otherwise there).

Reply:
A necessary decision here is whether 106 or 107 is better.  It would be nice to find prior art on that!

Reply:
I think it should be 106, but I'm not dogmatic about it. If you want, we can just ask if anybody wants it to be 107 on the mailing list, but I really think it's fine as it is, since the flaw with nmant is that it's off by 1, not 2.

Reply:
Here is the prior art: https://www.ibm.com/docs/en/aix/7.2?topic=sepl-128-bit-long-double-floating-point-data-type